### DISCUSSION TITLE
Does this support emqx v5 ?

### DISCUSSION THREAD

**Discussion by codering** [emqx/emqx-operator#269] [Q&A] [answered]
I don't find the config example for emqx v5.

And the [doc](https://github.com/emqx/emqx-operator/blob/main/docs/zh_CN/configuration/configuration.md) is not working .

**Comment by Rory-Z** [comment]
Not yet, but will be soon

**Comment by Rory-Z** [comment]
> And the [doc](https://github.com/emqx/emqx-operator/blob/main/docs/zh_CN/configuration/configuration.md) is not working .

Sorry about this, this doc is not update, please see https://github.com/emqx/emqx-operator/blob/main/docs/en_US/reference/v1beta3-reference.md

**Reply by codering** [reply]
I see a new version released. How is it going on now for v5 ？

**Reply by Rory-Z** [reply]
Sorry, not yet
EMQX Operator 2.0.0-alpha.1 will come soon, in 2.0.0-alpha.1, we will support EMQX 5.0

**Comment by Rory-Z** [comment, marked as answer]
Hi， @codering EMQX Operator 2.0.0 Released, please see https://github.com/emqx/emqx-operator/blob/main/docs/en_US/getting-started/getting-started.md#deploy-emqx-5
